### PR TITLE
improve proceed pipeline but not required parameters

### PR DESCRIPTION
**Pull request by soulseen**
Signed-off-by: soulseen <sunzhu@yunify.com>

# Description

See [JENKINS-58406](https://issues.jenkins-ci.org/browse/JENKINS-58406).

# Submitter checklist
- [ ] Link to JIRA ticket in description, if appropriate.
- [ ] Change is code complete and matches issue description
- [ ] Appropriate unit or acceptance tests or explanation to why this change has no tests
- [ ] Reviewer's manual test instructions provided in PR description. See Reviewer's first task below.

# Reviewer checklist
- [ ] Run the changes and verified the change matches the issue description
- [ ] Reviewed the code
- [ ] Verified that the appropriate tests have been written or valid explanation given

### CODE REVIEW

File: blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/rest/impl/pipeline/PipelineStepImpl.java [hunk 1]
```diff
@@ -231,7 +227,15 @@ public HttpResponse submitInputStep(StaplerRequest request) {
...
             }
 
-            Object o = parseValue(execution, JSONArray.fromObject(body.get(PARAMETERS_ELEMENT)), request);
+            Object o;
+            InputStep input = execution.getInput();
+            if (body.get(PARAMETERS_ELEMENT) != null) {
+                System.out.println("use parameters");
+                o = parseValue(input, JSONArray.fromObject(body.get(PARAMETERS_ELEMENT)), request);
+            } else {
+                System.out.println("not use parameters");
```
Comment (halkeye) [inline blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/rest/impl/pipeline/PipelineStepImpl.java]: Can you get rid of the printlns?

Comment (soulseen) [reply blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/rest/impl/pipeline/PipelineStepImpl.java]: ok.

**Comment by bitwiseman** [comment]
Still no tests for this.

**Comment by bitwiseman** [comment]
Closing for now. Please update and we can reopen.

**Pull request closed without merging**